code-challenges.js

```diff
@@ -41,16 +71,24 @@ var vowelTester1 = "learn"
 var vowelTester2 = "throw"
 // Expected output: 3
 
+const firstVow =("") {
+var vowTestArr =  vowelTester1.split
+  for newArr (let i = 0; i < vowelTester1.length, i++){
+    char c = vowelTester1.charAt(i)
+    if (isVowel(c))
+      return vowelTester1.indexOf(c)
+  }
```

Comment:
I totally see where you were going with this and I think you had the right idea. Unfortunately this code won't run due to several syntax errors and the general formatting of how your for loop works etc.  If you'd like to go over this again, I'd love to be a resource to you at any time to help get these fundamentals down!